absl-flat_hash_map.hh

```diff
@@ -29,6 +38,10 @@ template <typename K, typename V, typename... Ts>
 struct flat_hash_map : public absl::flat_hash_map<K, V, Ts...> {
 };
 
+template <typename V>
+struct flat_hash_map<size_t, V>
+        : public absl::flat_hash_map<size_t, V, integral_hash<size_t>> {};
+
```

Comment:
I think we can use absl::Hash and absl::flat_hash_map directly. IIRC absl-flat_hash_map.hh was introduced to work around some integration problems with sstring, but here we can just use it directly.

Especially since the new flat_hash_map eats the key type parameter.

Reply:
Acked.